pkg/pwgen/pwgen_test.go

```diff
@@ -2,13 +2,14 @@ package pwgen
 
 import (
 	"bytes"
-	"crypto/rand"
+	crand "crypto/rand"
 	"fmt"
 	"io"
-	mrand "math/rand"
```

Comment:
This choice was questioned on orangekame's PR. 

Howver, this aligns `pwgen_test.go` with the style found in `rand.go`, of the same package.